spark/src/main/kotlin/com/adevinta/spark/components/bottomsheet/modal/ModalBottomSheet.kt

```diff
@@ -1,474 +0,0 @@
-import com.adevinta.spark.components.bottomsheet.ConsumeSwipeWithinBottomSheetBoundsNestedScrollConnection
-import com.adevinta.spark.components.bottomsheet.SheetState
-import com.adevinta.spark.components.bottomsheet.SheetValue
-import com.adevinta.spark.components.bottomsheet.SheetValue.Expanded
-import com.adevinta.spark.components.bottomsheet.SheetValue.Hidden
-import com.adevinta.spark.components.bottomsheet.SheetValue.PartiallyExpanded
-import com.adevinta.spark.components.bottomsheet.rememberSheetState
-import com.adevinta.spark.components.bottomsheet.swipeAnchors
-import com.adevinta.spark.components.buttons.ButtonFilled
-import com.adevinta.spark.components.icons.Icon
-import com.adevinta.spark.components.list.ListItem
-import com.adevinta.spark.components.surface.Surface
-import com.adevinta.spark.components.text.Text
-import com.adevinta.spark.components.toggles.CheckboxLabelled
-import com.adevinta.spark.icons.LikeFill
-import com.adevinta.spark.icons.SparkIcons
-import com.adevinta.spark.tokens.contentColorFor
-import kotlinx.coroutines.CoroutineScope
-import kotlinx.coroutines.launch
-import kotlin.math.max
-
-/**
- * <a href="https://m3.material.io/components/bottom-sheets/overview" class="external" target="_blank">Material Design modal bottom sheet</a>.
- *
- * Modal bottom sheets are used as an alternative to inline menus or simple dialogs on mobile,
- * especially when offering a long list of action items, or when items require longer descriptions
- * and icons. Like dialogs, modal bottom sheets appear in front of app content, disabling all other
- * app functionality when they appear, and remaining on screen until confirmed, dismissed, or a
- * required action has been taken.
- *
- * ![Bottom sheet image](https://developer.android.com/images/reference/androidx/compose/material3/bottom_sheet.png)
- *
- * A simple example of a modal bottom sheet looks like this:
- *
- * @sample androidx.compose.material3.samples.ModalBottomSheetSample
- *
- * @param onDismissRequest Executes when the user clicks outside of the bottom sheet, after sheet
- * animates to [Hidden].
- * @param modifier Optional [Modifier] for the bottom sheet.
- * @param sheetState The state of the bottom sheet.
- * @param shape The shape of the bottom sheet.
- * @param containerColor The color used for the background of this bottom sheet
- * @param contentColor The preferred color for content inside this bottom sheet. Defaults to either
- * the matching content color for [containerColor], or to the current [LocalContentColor] if
- * [containerColor] is not a color from the theme.
- * @param tonalElevation The tonal elevation of this bottom sheet.
- * @param scrimColor Color of the scrim that obscures content when the bottom sheet is open.
- * @param dragHandle Optional visual marker to swipe the bottom sheet.
- * @param content The content to be displayed inside the bottom sheet.
- */
-@Composable
-@ExperimentalMaterial3Api
-public fun ModalBottomSheet(
```

Comment:
This is a breaking change, this should be deprecated instead as it's used 30+ times in Polaris